proposals/specification/agent-management/ard-request.bs

```diff
@@ -0,0 +1,24 @@
+
+### Discovery Request ### {#sec-ard-request}
+
+Once the [=Agent Manager=] of an [=Entity=] is known, an [=Agent=] wishing to
+discover its [=Agent-Specific Resources=] related to that [=Entity=] [SHOULD]
+query the [=Agent Manager=] by sending a [:HEAD:] or [:GET:] request to the
+[=Agent Manager's=] URL in which the value of the [:Authorization:] header is
+set to a [[DPOP]]-bound [[OIDC]] [=Identity Token=] containing an [=Identifier=]
+of the [=Agent=] as value of the `azp` [=Claim=] or as *sole* value of the
+`aud` [=Claim=].
```

Comment:
If we want to be that specific we should probably reference Solid OIDC.
The ID Token is only pushed as a claim and access token should be used in Authorization header.

Reply:
You're right! This should be an Access Token with the `client_id` set to the URI of the Agent.

Reply:
Solid OIDC doesn't put any more specific requirement on the format of Access Tokens, they are left up to RS & AS.
I think we should use some abstract language and have non-normative reference to Solid-OIDC as a concrete example.

Reply:
@csarven we think this should be discussed on Wednesday

Reply:
ACTION: remove token details, talk about "authorized request" and refer to (yet to be created) solid ecosystem options